<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>0</class>
 <widget class="QMainWindow" name="0">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="seq_view_analysis_label">
      <property name="text">
       <string>Select nucleotide sequence view:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="radiobtn_layout">
      <item>
       <widget class="QRadioButton" name="seq_radiobtn">
        <property name="text">
         <string>Main Sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="comp_radiobtn">
        <property name="text">
         <string>Complement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rev_comp_radiobtn">
        <property name="text">
         <string>Reverse Complement</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="whichseq_analysis_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <layout class="QHBoxLayout" name="show_nuc_layout">
      <item>
       <widget class="QLabel" name="show_nuc_label">
        <property name="text">
         <string>Show nucleotides</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="show_nuc_start_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="to_label">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="show_nuc_end_edit"/>
      </item>
     </layout>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QTextBrowser" name="seq_analysis_txtbrwsr"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="nucfreq_analysis_txtbrwsr"/>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="dinucfreq_analysis_txtbrwsr"/>
    </item>
    <item row="5" column="0">
     <widget class="QTextBrowser" name="gc_cont_analysis_txtbrwsr"/>
    </item>
    <item row="7" column="0">
     <widget class="QTextBrowser" name="length_analysis_txtbrwsr"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="length_analysis_label">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="gc_cont_analysis_label">
      <property name="text">
       <string>GC Content</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dinucfreq_analysis_label">
      <property name="text">
       <string>Dinucleotide Frequencies</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nucfreq_analysis_label">
      <property name="text">
       <string>Nucleotide Frequencies</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionsave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All_Analysis"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close Current Analysis</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Analysis</string>
   </property>
  </action>
  <action name="actionClose_All_Analysis">
   <property name="text">
    <string>Close All Analysis</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
